### PR TITLE
RELATED: RAIL-3587 make the How to use more prominent

### PR DESCRIPTION
**Pull request by no23reason**
Previously, the most important link to use was drowned in a wall of text.
This adds headers so the user can scan the page more efficiently
if they don't care about the details described above.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/785/ : SUCCESS in 1m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/502/ : SUCCESS in 7s

File: docs/02_start__using_boilerplate.md [hunk 1]
```diff
@@ -24,6 +26,8 @@ This is what is going to happen when you run `@gooddata/create-gooddata-react-ap
 3. Once you confirm your domain, the tool will install required dependencies and create the structure of your application.
 4. It will then prompt you to start your application and will open the welcome page in your browser.
 
+## How to use
```
Comment (evgeniabogranskaya) [inline docs/02_start__using_boilerplate.md:29]: How to use `@gooddata/create-gooddata-react-app`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/787/ : SUCCESS in 1m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/504/ : SUCCESS in 6s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR438

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/789/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/139/ : SUCCESS in 36s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/39/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/229/ : SUCCESS in 18s

**Pull request merged**